cmake/FindPoly.cmake

```diff
@@ -143,6 +143,17 @@ if(NOT Poly_FOUND_SYSTEM)
       "${DEPS_BASE}/lib/libpicpolyxx${CMAKE_STATIC_LIBRARY_SUFFIX}")
   endif()
 
+  # If it isn't a wasm compilation
+  if(WASM STREQUAL "OFF")
+    set(POLY_PATCH_CMD_WASM "")
```

Comment:
I think that it would be cleaner/easier to just append to `POLY_PATCH_CMD` as needed instead of introducing `POLY_PATCH_CMD_WASM`.